SPECS/xcp-ng-generic-lib.spec

```diff
@@ -55,5 +60,11 @@ This package provides documentation and development headers for xcp-ng-generic-l
 %{_libdir}/libxcp-ng-generic.so
 
 %changelog
+* Tue May 28 2019 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.1.0-1
+- Rename xcp_select to xcp_poll
```

Comment:
The RPM changelog should generally be written for users, so no implementation details.

Also, doesn't this version fix PV migration?

Reply:
Oops, this is generic lib, not xcp-emu-manager. Well, then I don't know :)